plugins/modules/content_export_version.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2022, Jeremy Lenz <jlenz@redhat.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: content_export_version
+version_added: 3.5.0
+short_description: Manage pulp3 content exports
+description:
+    - Export a content view version to a directory.
+author:
+    - "Jeremy Lenz (@jeremylenz)"
+options:
+  id:
+    description:
+      - Content view version identifier.
+    required: true
+    type: int
```

Comment:
I don't think the user should have to pass the cv-id, but more the cv name, and then the module should figure this out on its own.